File: gastroticket-backend/src/main/java/uoc/tfm/gastroticket/empleados/model/EmpleadoRequest.java
package uoc.tfm.gastroticket.empleados.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@AllArgsConstructor
@NoArgsConstructor
@Data
public class EmpleadoRequest {
    private long idEmpleado;
    private long idUsuario;
    private long idEmpresa;
}

File: gastroticket-frontend/src/app/Models/empleadoRequest.ts
export class EmpleadoRequest{
    idEmpleado: number;
    idUsuario: number;
    idEmpresa: number;

    constructor(idEmpleado: number, idUsuario: number, idEmpresa: number){
        this.idEmpleado = idEmpleado;
        this.idUsuario = idUsuario;
        this.idEmpresa = idEmpresa;
    }
}